test/integration/test-form-get-length.js

```diff
@@ -90,4 +91,34 @@ var fs = require('fs');
   form.getLength(callback);
 })();
 
+(function testReadableStreamData() {
+  var form = new FormData();
+  var expectedLength = 0;
 
+  var util = require('util');
+  util.inherits(CustomReadable, Readable);
+
+  function CustomReadable(opt) {
+    Readable.call(this, opt);
+    this._max = 2;
+    this._index = 1;
+  }
+
+  CustomReadable.prototype._read = function() {
+    var i = this._index++;
+    if (i > this._max)
+      this.push(null);
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Expected '{' and instead saw 'this'.](https://www.codacy.com/app/form-data/form-data/file/875565513/issues/source?bid=2805153&fileBranchId=2805172#l110)